pkg/mods/str/str.d.elv

```diff
@@ -54,6 +54,19 @@ fn count {|str substr| }
 # ```
 fn equal-fold {|str1 str2| }
 
+
+# Splits `$str` around each instance of one or more consecutive white space
+# characters.
+#
+# ```elvish-transcript
+# ~> str:split "lorem ipsum   dolor"
+# ▶ lorem
+# ▶ ipsum
+# ▶ dolor
+# ~> str:split "   "
+# ```
```

Comment:
Add a "See also" line to str:split and vice-versa. That will make it easier to discover.